docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "2.2"
+services:
+  database:
+    image: metaskills/mssql-server-linux-rails
+    ports:
+      - "1433:1433"
+  tests:
+    build: .
+    command: wait-for database:1433 -- bundle exec rake test ACTIVERECORD_UNITTEST_HOST=database
```

Comment:
Can `ACTIVERECORD_UNITTEST_HOST=database` be moved into the service definition?

Reply:
Any special reason to do that? Just trying to understand...

Reply:
it's pretty conventional with docker compose to put env vars outside the command, if only to itemize them & simplify your command syntax.

Reply:
Yeah, you're right. I first read image definition, instead of service definition. I'm out on holidays and will fix that as soon as I'm back.

Reply:
That's done now.